pkg/catalog/generator.go

```diff
@@ -44,8 +43,8 @@ type Generator struct {
 	Fs                   afero.Fs
 	Module               *sysl.Module
 	errs                 []error // Any errors that stop from rendering will be output to the browser
-	ProjectTempl         *template.Template
-	PackageTempl         *template.Template
+	Templates            []*template.Template
```

Comment:
I think a comment here would help lots, specifying that the first item is always the project template and the second item is always the package template

Reply:
Not "Always" any order could be valid